pkg/autoscaler/statforwarder/forwarder.go

```diff
@@ -49,27 +53,43 @@ const (
 	retryInterval      = 100 * time.Millisecond
 )
 
+// statProcessor is a function to process a single StatMessage.
+type statProcessor func(sm asmetrics.StatMessage)
+
+// bucketProcessor includes the inforamtion about how to process
+// the StatMessage owned by a bucket.
+type bucketProcessor struct {
+	// holder is the HolderIdentity for a bucket from the Lease
+	holder string
+	// conn is the WebSocket connection to the holder pod.
+	conn *websocket.ManagedConnection
+	// proc is the function to process the StatMessage owned by the bucket.
+	proc statProcessor
+}
+
 // Forwarder does the following things:
 // 1. Watches the change of Leases for Autoscaler buckets. Stores the
 //    Lease -> IP mapping.
 // 2. Creates/updates the corresponding K8S Service and Endpoints.
-// 3. Can be used to forward the metrics falling in a bucket based on
-//    the holder IP. (This is a TODO)
+// 3. Can be used to forward the metrics owned by a bucket based on
+//    the holder IP.
 type Forwarder struct {
 	selfIP          string
 	logger          *zap.SugaredLogger
 	kc              kubernetes.Interface
 	endpointsLister corev1listers.EndpointsLister
+	// accept is the function to process a StatMessage which doesn't need
```

Comment:
```suggestion
	// `accept` is the function to process a StatMessage which doesn't need
```
may be `processor` and `forwarders`?